mail/postfix/src/opnsense/service/templates/OPNsense/Postfix/main.cf

```diff
@@ -43,9 +43,9 @@ smtp_tls_CAfile = /etc/ssl/cert.pem
 ##########################
 
 {% if helpers.exists('OPNsense.postfix.general.myhostname') and OPNsense.postfix.general.myhostname != '' %}
-myhostname = {{ OPNsense.postfix.general.myhostname }}
+myhostname = {{ OPNsense.postfix.general.myhostname }}.{{ OPNsense.postfix.general.mydomain }}
 {% else %}
-myhostname = {{ system.hostname }}
+myhostname = {{ system.hostname }}.{{ system.domain }}
```

Comment:
```suggestion
myhostname = {{ system.hostname }}.$mydomain
```

this is safer I think if postfix will have it.. the user could always change the postfix domain setting and then we take the one that is currect